package.json

```diff
@@ -22,37 +20,26 @@
     "base64-arraybuffer": "0.2.0",
     "bech32": "2.0.0",
     "bignumber.js": "9.0.1",
-    "bip32": "2.0.6",
     "bip39": "3.0.2",
-    "bn.js": "5.2.0",
     "browser-passworder": "2.0.3",
-    "bs58check": "2.1.2",
     "classnames": "2.2.6",
-    "elliptic": "6.5.4",
     "extensionizer": "1.0.1",
     "i18next": "19.8.4",
-    "identicon.js": "^2.3.3",
-    "js-sha256": "0.9.0",
-    "js-sha512": "0.8.0",
-    "node-sass": "4.14.1",
     "obs-store": "4.0.3",
     "prop-types": "15.7.2",
     "qrcode-generator": "1.4.4",
     "react": "17.0.1",
     "react-circular-progressbar": "2.0.3",
-    "react-compound-slider": "3.3.1",
     "react-dom": "17.0.1",
     "react-helmet": "6.1.0",
     "react-i18next": "11.8.2",
     "react-idle-timer": "4.2.5",
     "react-redux": "7.2.2",
-    "react-router": "5.1.2",
     "react-router-dom": "5.1.2",
-    "react-scripts": "4.0.1",
     "redux": "4.0.5",
     "safe-buffer": "5.2.1",
     "valid-url": "1.0.9",
-    "webpack": "4.44.2"
+    "tweetnacl": "1.0.3"
```

Comment:
Please run e.g. `yarn upgrade tweetnacl` so that dependencies get alphabetically sorted

Reply:
ok , i will update this